sections/package.zsh

```diff
@@ -29,7 +29,7 @@ spaceship_package() {
   spaceship::exists npm || return
 
   # Grep and cut out package version
-  local package_version=$(grep -E '"version": "v?([0-9]+\.){1,}' package.json | cut -d\" -f4 2> /dev/null)
+  local package_version=$(grep -E '^  "version": "v?([0-9]+\.){1,}' package.json | cut -d\" -f4  2>/dev/null)
```

Comment:
What if person use tabs in their `package.json`? Can we use `[ \t]`?

Reply:
So two tabs before specifying version ?

Reply:
Just `^\s+"version"...` might work as well

Reply:
The goal is to reduce the false positive rate (see original issue). `\s+` will have (almost) the same false positive rate.

I suggested 2 spaces because that's what npm uses when it autoformats `package.json`. I understand the "package" section is intended to cover more than just nodejs modules, but I suspect other ecosystems have followed npm's convention for indentation (but correct me if a target ecosystem diverges).

Of course this is not perfect, as npm will accept JSON regardless of indentation. But hand-formatted JSON doesn't seem to be the common case.

Reply:
Oh I see, in this case my vote is to go with `jq` and properly parse JSON 🙂